lib/adapter.js

```diff
@@ -5662,6 +5662,18 @@ function Adapter(options) {
          */
         this.sendToHostAsync = tools.promisifyNoError(this.sendToHost, this);
 
+        /**
+         * Send notification with given scope and category to host of this adapter
+         *
+         * @param {string} scope - scope to be addressed
+         * @param {string|null} category - to be addressed, if null message will be checked by regex of given scope
+         * @param {string} message - message to be stored/checked
+         * @return Promise<void>
+         */
+        this.notifyHost = async (scope, category, message) => {
```

Comment:
notifyHost implies that we notify the host about something (but target for the host)... maybe sendNotification(ToHost) or registerNotification?